requirements.txt

```diff
@@ -18,4 +18,5 @@ tensorflow-privacy~=0.0.1
 # TODO(b/141279425): Remove pinned tf-estimator-nightly version.
 tf-estimator-nightly==1.14.0.dev2019091601
 tf-nightly
+grpcio~=1.22.0
```

Comment:
If you pull latest and resolve conflicts, you would see that we are pinning to grpcio 1.22.0 already (deps are alphabetized so you might not actually get a conflict).